pom.xml

```diff
@@ -162,23 +162,23 @@
         <test.xml.parser>org.eclipse.persistence.platform.xml.jaxp.JAXPParser</test.xml.parser>
 
         <!--Eclipse Dependencies version-->
-        <activation.version>2.0.0</activation.version>
+        <activation.version>2.0.1</activation.version>
```

Comment:
don't update this one yet; it's in staging only for now

Reply:
OK reverted back to 2.0.0